CoordgenFragmenter.cpp

```diff
@@ -149,16 +149,16 @@ void CoordgenFragmenter::initializeInformation(
 bool CoordgenFragmenter::setFixedInfo(sketcherMinimizerFragment* fragment)
 {
     fragment->fixed =
-        (find_if(fragment->atoms().begin(), fragment->atoms().end(),
-                 isAtomFixed) != fragment->atoms().end());
+        (count_if(fragment->atoms().begin(), fragment->atoms().end(),
```

Comment:
I believe it is possible for fragments to have a single atom, in which case we would likely want to label the fragment as constrained/fixed